eternalcore-core/src/main/java/com/eternalcode/core/feature/teleport/request/TpaCommand.java

```diff
@@ -46,14 +48,24 @@ void execute(Player player, @Arg Player target) {
             .placeholder("{PLAYER}", target.getName())
             .send();
 
-        this.noticeService
-            .create()
-            .player(target.getUniqueId())
-            .notice(translation -> translation.tpa().tpaReceivedMessage())
-            .placeholder("{PLAYER}", player.getName())
-            .send();
+        this.isIgnoring(target, player).then((isIgnoring) -> {
+            if (isIgnoring) {
+                return;
+            }
+
+            this.noticeService
+                .create()
+                .player(target.getUniqueId())
+                .notice(translation -> translation.tpa().tpaReceivedMessage())
+                .placeholder("{PLAYER}", player.getName())
+                .send();
+        });
 
         this.requestService.createRequest(player.getUniqueId(), target.getUniqueId());
```

Comment:
I think you should create request when target is not ignored

Reply:
That its works

Reply:
Powinieneś stworzyć requesta tylko wtedy kiedy target nie jest ignorowany.
Obecnie "zasłaniasz" wiadomość ale tak po za tym zawsze wysyłasz tpa requesta, co może być problematyczne jak ktoś wpisze `/tpaaccet` i nieświadomie zaakceptuje ostatniego "ignorowanego" requesta (a np. spodziewał się akceptacji starszego requesta którego widział na chat)